Disabled CA verification on get requests for CA-SK

Nil-Cipher opened the pull request:
## Issue

<!-- If you want to close an issue automatically when your PR is merged, write "Closes X" where X is the PR number. For example: Closes #000 -->
Closes #5562



## Description

<!-- Explains the goal of this PR -->
Disables CA verification for the GET requests on the Saskatchewan data sites.

### Preview

<!-- Please add screenshots and/or gif that shows visual changes (if applicable) -->

### Double check

- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [ ] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

jarek commented on parsers/CA_SK.py:
```diff
@@ -115,7 +115,9 @@ def fetch_consumption(
     headers = {"user-agent": USER_AGENT}
 
-    response: Response = session.get(CONSUMPTION_URL)  # , headers=headers)
+    response: Response = session.get(
+        CONSUMPTION_URL, verify=False
+    )  # , headers=headers)
```
It's cleaner if you remove the dummy comment

```suggestion
    response: Response = session.get(CONSUMPTION_URL, verify=False)
```

Nil-Cipher replied:
done

VIKTORVAV99 commented on parsers/CA_SK.py:
```diff
@@ -115,7 +115,7 @@ def fetch_consumption(
     headers = {"user-agent": USER_AGENT}
 
-    response: Response = session.get(CONSUMPTION_URL)  # , headers=headers)
+    response: Response = session.get(CONSUMPTION_URL, verify=False)
```
```suggestion
    response: Response = session.get(CONSUMPTION_URL, headers=headers)
```

VIKTORVAV99 commented on parsers/CA_SK.py:
```diff
@@ -64,7 +64,7 @@ def fetch_production(
     # Set the headers to mimic a user browser as the API will return a 403 if not.
     headers = {"user-agent": USER_AGENT}
-    response: Response = session.get(PRODUCTION_URL, headers=headers)
+    response: Response = session.get(PRODUCTION_URL, headers=headers, verify=False)
```
```suggestion
    response: Response = session.get(PRODUCTION_URL, headers=headers)
```

VIKTORVAV99 reviewed (changes requested):
Looks like the SSL cert is back up and valid.
However we are getting 403 errors due to the user agent being disallowed.

I have added suggestions that should resolve this.

Nil-Cipher commented:
> Looks like the SSL cert is back up and valid. However we are getting 403 errors due to the user agent being disallowed.
> 
> I have added suggestions that should resolve this.

Added suggested changes.

VIKTORVAV99 reviewed (approved):
LGTM! 🎉

(But it looks like the data is unavailable now...)
